io.github.cybertimon.rapidraw.yml

```diff
@@ -0,0 +1,58 @@
+id: io.github.cybertimon.rapidraw
+
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+
+command: RapidRAW
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: binary
+    buildsystem: simple
+
+    sources:
+      - type: file
+        path: io.github.cybertimon.rapidraw.metainfo.xml
+      - type: file
+        url: https://github.com/CyberTimon/RapidRAW/releases/download/v1.3.10/03_RapidRAW_v1.3.10_ubuntu-24.04_amd64.deb
```

Comment:
This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source